src/lib/server/postgres_protocol_handler.hpp

```diff
@@ -64,6 +65,9 @@ class PostgresProtocolHandler {
   // Send error message to client if there is an error during parsing or execution
   void send_error_message(const ErrorMessage& error_message);
 
+  // Send warning message to client if there is an error during parsing or execution
+  void send_warning_message(const WarningMessage& warning_message);
```

Comment:
It was supposed to be consistent the the ErrorMessage tyoe. However I changed it to std::string, since this has been a discussion among our group before.